ElectronClient/gui/MainScreen.jsx

```diff
@@ -194,6 +194,29 @@ class MainScreenComponent extends React.Component {
 					},
 				},
 			});
+		} else if (command.name === 'moveToFolder') {
+			const folderItems = this.props.folders;
+			const startFolders = folderItems
+				.map(a => {
+					return { key: a.id, value: a.id, label: a.title };
+				});
+
+			this.setState({
+				promptOptions: {
+					label: _('Move to Notebook:'),
```

Comment:
Sorry could you make it lower case? "Move to notebook:" It's for consistency with the rest of the app.

Reply:
Done.

Reply:
Move to notebook